content/blog/2024/10/03/2024-10-03-jenkins-election-candidates.adoc

```diff
@@ -0,0 +1,223 @@
+link:https://community.jenkins.io/u/stefanspieker/summary[Jenkins community],
+link:https://stackoverflow.com/users/1287864/s-spieker[Stackoverflow]
+
+[[jonesbusy]]
+=== link:/blog/authors/jonesbusy/[Valentin Delaye]
+
+I've been a Jenkins user since it's inception in 2011 and contributor since 2018.
+As of today I maintain more than 35 plugins.
+
+Starting of this year I got even more involved in the Jenkins community, by participating as a mentor for the Google Summer of Code to work on the Jenkins Plugin Modernizer Tool, which is a CLI to update plugins at scale using OpenRewrite.
+
+I'm particularly interested in the "cloud ready" aspect of Jenkins including deployments, packaging (like Jenkinsfile Runner), or distributed storage.
+A recent contribution on this topic is the link:https://plugins.jenkins.io/artifactory-artifact-manager/[Artifactory Artifact Manager plugin] that I authored and maintain.
+
+If I'm elected on the Jenkins governance board those will be my subjects of choice.
+
+Affiliations: ELCA Cloud Services
+
+Profile Links:
+link:https://github.com/jonesbusy[GitHub],
+link:https://www.linkedin.com/in/valentindelaye/[LinkedIn],
+link:https://contributors.jenkins.io/pages/contributors/valentin-delaye/[Contributor spotlight],
+link:https://community.jenkins.io/u/jonesbusy/summary[Jenkins community]
+
+[[krisstern]]
+=== link:/blog/authors/krisstern/[Kris Stern]
+
+I have been a Jenkins contributor since 2021.
+Currently I am a maintainer of four plugins, have been a GSoC org admin and mentor for Jenkins for three years, and continue to be a Jenkins Release Team member.
+Academically I have been trained as an observational astrophysicist and obtained my PhD degree from the University of Hong Kong (HKU) in 2021.
+I have worked professionally as a software engineer since late 2019, currently working in a AiFi startup.
+Currently I am also a part-time MCIT Online student at UPenn.
+I am passionate about open source and would like to share my love of open-source software with the others.
+I feel grateful to be nominated. If elected, I would love to bring with me a startup mindset to help keep Jenkins perpetually youthful and relevant as well as a welcoming community to newcomers.
+
+Affiliations: GAIB.ai
+
+Profile Links:
+link:https://github.com/krisstern[GitHub],
+link:https://www.linkedin.com/in/kris-stern/[LinkedIn],
+link:https://contributors.jenkins.io/pages/contributors/kris-stern/[Contributor spotlight],
+link:https://community.jenkins.io/u/krisstern/summary[Jenkins community]
+
+[[oleg_nenashev]]
+=== link:/blog/authors/oleg_nenashev/[Oleg Nenashev]
+
+Jenkins core maintainer, open source software and open hardware advocate.
+Oleg started using Hudson for Hardware/Embedded projects in 2008 and became an active Jenkins contributor in 2012.
+He maintains Jenkinsfile Runner and organizes Jenkins meetups in Switzerland and online.
+
+Affiliations: Gradle, Inc.
```

Comment:
```suggestion
I’ve participated in Jenkins since 2012, including being a core maintainer since 2014,
I was a governance board member from 2019 for two terms (Dec 2019-2023) and also represented Jenkins on the Continuous Delivery Foundation TOC and the board.
During my work on Jenkins, I participated in many projects including JCasC, pluggable storage, and Jenkinsfile Runner. I also created or maintained around 30 plugins for Jenkins.
Many community programs, including GSoC and Hacktoberfest participation, were initially started by me.
In 2022. I took a sabbatical as a core and plugin maintainer due to the war in Ukraine, but I plan to return to the maintainer roles, too.

For me, Jenkins has one of the strongest open source communities, and I am proud to be a part of it.
Should I be elected to the board, my focus will be community growth via onboarding new contributors, improving contributor/developer experience for Jenkins, and facilitating technical partnerships.
The ecosystem evolves, and Jenkins should be changing and growing along with that.
For me, the priorities would be adopting open standards,
integrations with the key projects and services in the cloud-native, CI/CD, and Java space, and continuing Jenkins' evolution as a multi-purpose automation framework.

Affiliations: Gradle Inc., CNCF/CDF, API Neuchatel, Independent
```